docs/index.md

```diff
@@ -1,14 +1,14 @@
 ## Tutorial: Building an internet connected lighting system
 
-In this tutorial, you'll use the ARM Mbed IoT Device Platform to build a complete connected lighting system that you can deploy in your house or office. The system consists of one (or more) extra-bright RGB LEDs hooked up to an Mbed OS development board. A motion sensor can turn the lights on and off. The lights are also connected to the internet, so you can change their color. The system uses Mbed Cloud to connect to the internet. It is secured end-to-end, and you can update it through over-the-air firmware updates.
+In this tutorial, you'll use the ARM Mbed IoT Device Platform to build a complete connected lighting system that you can deploy in your house or office. The system consists of one (or more) extra-bright RGB LEDs hooked up to an Mbed OS development board. A motion sensor can turn the lights on and off. The lights are also connected to the internet, so you can change their color. The system uses Device Management to connect to the internet. It is secured end-to-end, and you can update it through over-the-air firmware updates.
```

Comment:
The thing is, we called it Arm Mbed IoT Device Platform and now two lines down it's Device Management (which is also a concept) without explicitly saying it's the name of the product. Of course, it's in the context of the whole docs suite, so maybe it's okay?

Reply:
Hang on, that should be a 'Pelion IoT Platform'.